protos/component_metadata_server/component_metadata_server.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package mavsdk.rpc.component_metadata_server;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.component_metadata_server";
+option java_outer_classname = "ComponentMetadataServerProto";
+
+// Provide component metadata json definitions, such as parameters.
+service ComponentMetadataServerService {
+    /*
+     * Provide metadata (can only be called once)
```

Comment:
Why only once? Does that mean it can't change?

Reply:
There's a protocol extension proposal for dynamic changes, but for now it's not possible. In the majority of cases it's also not needed.

Reply:
So hang on, but you have to call it multiple times for the different metadata types, right?

Reply:
No, the metadata is provided as an array: `repeated Metadata metadata`